app/serializers/country_bread_serializer.rb

```diff
@@ -0,0 +1,42 @@
+class CountryBreadSerializer
+  def initialize(country, country_breads)
+    @country = country
+    @breads = country_breads
+  end
+
+  def serialized_response
+    {
+      country: {
+        data: {
+          id: @country.id,
+          attributes: {
+            name: @country.name,
+            overview: @country.overview,
+            culinary: @country.culinary,
```

Comment:
What a fancy handrolled JSON response! Also removes those relationship hashes that weren't needed for FE.